test/parallel/test-path-parse-format.js

```diff
@@ -25,20 +25,27 @@ const assert = require('assert');
 const path = require('path');
 
 const winPaths = [
-  'C:\\path\\dir\\index.html',
-  'C:\\another_path\\DIR\\1\\2\\33\\\\index',
-  'another_path\\DIR with spaces\\1\\2\\33\\index',
-  '\\foo\\C:',
-  'file',
-  '.\\file',
-  'C:\\',
-  '',
+  // [path, root]
+  ['C:\\path\\dir\\index.html', 'C:\\'],
+  ['C:\\another_path\\DIR\\1\\2\\33\\\\index', 'C:\\'],
+  ['another_path\\DIR with spaces\\1\\2\\33\\index', ''],
+  ['\\foo\\C:', '\\'],
```

Comment:
I don't mean we should validate it, I just don't think we should test against invalid paths.